pom.xml

```diff
@@ -86,23 +88,29 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-api</artifactId>
-            <version>2.8</version>
+            <version>2.11</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.jenkins-ci.plugins</groupId>
+                    <artifactId>scm-api</artifactId>
```

Comment:
Avoid what? I am asking to pick up the `scm-api` dependency from some other dependency trail, not this one.